src/main/java/uk/gov/companieshouse/company_appointments/model/data/DeltaOfficerData.java

```diff
@@ -166,6 +166,11 @@ public DeltaOfficerData setCountryOfResidence(String countryOfResidence) {
     return this;
   }
 
+  public DeltaOfficerData setIsSecureOfficer(Boolean isSecureOfficer) {
```

Comment:
A setter already exists with name ```setSecureOfficer``` so could it be used in the unit test instead of creating this one please?

Reply:
I missed that setter. Done